plinth/modules/shaarli/views.py

```diff
@@ -28,6 +28,9 @@
 from plinth import package
 from plinth.modules import shaarli
 
+def on_install():
+"""Notify that the service is now enabled."""
+shaarli.service.notify_enabled(None, True)
 
 @package.required(['shaarli'])
```

Comment:
@phylophyl I'm not sure I follow...where is that screenshot from? Also...why doesn't it have the on_install function that you added in this pull request?